nginx/requirements.txt

```diff
@@ -0,0 +1 @@
+# integration pip requirements
```

Comment:
@olivielpeau and I had a discussion about this last week. He thought it was best to not add requirements if they are always going to be in the core agent. I agree with him but am open to this discussion: I don't know what's best.